project/project/CakePlatformDependencies.scala

```diff
@@ -99,21 +98,21 @@ object CakePlatformDependencies {
 
   object SbtDependencies {
     val buildInfo: ModuleID = "com.eed3si9n" % "sbt-buildinfo" % "0.7.0"
-    val digest: ModuleID = "com.typesafe.sbt" % "sbt-digest" % "1.1.3"
-    val dynver: ModuleID = "com.dwijnand" % "sbt-dynver" % "1.3.0"
+    val digest: ModuleID = "com.typesafe.sbt" % "sbt-digest" % "1.1.4"
+    val dynver: ModuleID = "com.dwijnand" % "sbt-dynver" % "2.0.0"
     val gatling: ModuleID = "io.gatling" % "gatling-sbt" % "2.2.2"
     val git: ModuleID = "com.typesafe.sbt" % "sbt-git" % "0.9.3"
     val gzip: ModuleID = "com.typesafe.sbt" % "sbt-gzip" % "1.0.2"
-    val header: ModuleID = "de.heikoseeberger" % "sbt-header" % "3.0.2"
+    val header: ModuleID = "de.heikoseeberger" % "sbt-header" % "4.1.0"
     val packager: ModuleID =
-      "com.typesafe.sbt" % "sbt-native-packager" % "1.3.1"
-    val pgp: ModuleID = "com.jsuereth" % "sbt-pgp" % "1.0.1"
+      "com.typesafe.sbt" % "sbt-native-packager" % "1.3.2"
+    val pgp: ModuleID = "com.jsuereth" % "sbt-pgp" % "1.1.0"
     val scalafmt: ModuleID = "com.lucidchart" % "sbt-scalafmt-coursier" % "1.15"
     val scalastyle: ModuleID =
       "org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0"
     val scoverage: ModuleID = "org.scoverage" % "sbt-scoverage" % "1.5.1"
-    val sonatype: ModuleID = "org.xerial.sbt" % "sbt-sonatype" % "2.0"
-    val wartRemover: ModuleID = "org.wartremover" % "sbt-wartremover" % "2.1.1"
+    val sonatype: ModuleID = "org.xerial.sbt" % "sbt-sonatype" % "2.1"
+    val wartRemover: ModuleID = "org.wartremover" %% "sbt-wartremover" % "2.2.1"
```

Comment:
We should plan on incorporating `scalafix` if we do this, as `wartremover` is being deprecated in favor of `scalafix`.

Reply:
I agree but would do in another PR